metabase/metabase
# Convert `admin.module.css` to CSS modules

Epic #38811

### Description

Converts standard CSS classes to CSS modules within `admin.module.css`. These classes affect the components within the `Admin` section of the app.

### How to verify

_will write down specific areas to look at_

**_Ensure that nothing looks weird (or surprises you), and that things haven't changed in these areas:_**

_General_
- The left panes within the Admin section of metabase
- Inputs for emails, passwords, and text. Make sure that text sizes, inputs, and the like are consistent.

_Route specific_
`/admin/databases`
- Database list
![image](https://github.com/metabase/metabase/assets/25306947/4cf3ce19-ef16-4dcf-ab78-10f58085bccd)

`/admin/datamodel/[segments AND metrics]`
- Actions menu on the right side of the row
![image](https://github.com/metabase/metabase/assets/25306947/1885ffa4-fea1-4bf1-bd9b-1f6d3dac4c09)
![image](https://github.com/metabase/metabase/assets/25306947/19e825af-fa73-449f-94c8-60df10ab4f0e)

`/admin/settings/maps`
- `Add a new map` widget (ignore the red border, this PR didn't create the problem. It'll be fixed in another PR)
![image](https://github.com/metabase/metabase/assets/25306947/c771f1a1-ef63-4855-93a8-ecff582ea9f8)

`/admin/people`
- the list of users

`/admin/people/groups`
- the popover for a row in groups
![image](https://github.com/metabase/metabase/assets/25306947/b584b65c-4ee9-4995-a7b9-54d37824a309)

`/admin/settings/general`
- the site url widget 
![image](https://github.com/metabase/metabase/assets/25306947/dc8133d2-3328-447d-b063-7a5710ee707d)

`/admin/troubleshooting/jobs`
- the left pane
- the modal that appears when clicking `View triggers` on a job
![image](https://github.com/metabase/metabase/assets/25306947/70764f8d-8b7e-4928-ba71-998fb5a89756)

`/admin/settings/authentication/api-keys`
- the API key table

`/admin/settings/authentication/[ldap AND saml AND jwt]`
- the group schema / mapping tables
![image](https://github.com/metabase/metabase/assets/25306947/a7203c42-58b5-48f8-b936-6660e75071bf)

`/admin/datamodel/database/`
- Almost all of it - play around and see if you can find anything weird.

	
`/admin/tools/model-caching`
- the table

`/admin/settings/public-sharing`
- the table of public links

`/admin/troubleshooting/tasks`
- the table
